### PR TITLE
(chore) Getting started tweaks

### PR DESCRIPTION
**Pull request by yyyc514**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,21 +130,48 @@ For more info about the returned object refer to the api docs https://highlightj
...
-hljs.registerLanguage('sql', require('highlight.js/lib/languages/sql'));
-// highlight with providing the language
+
 const highlightedCode = hljs.highlight('html', '<span>Hello World!</span>').value
 ```
 
+
+## ES6 Modules
+
+First you'll likely install via `npm` -- see [Getting the Library](#getting-the-library) below.
```
Comment (allejo) [inline README.md]:
```suggestion
First, you'll likely install via `npm` or `yarn` -- see [Getting the Library](#getting-the-library) below.
```

File: README.md [hunk 2]
```diff
@@ -153,64 +180,95 @@ both AMD and CommonJS, so if you wish you can use RequireJS or
...
 
-```js
-import hljs from 'highlight.js/lib/core';
-import javascript from 'highlight.js/lib/languages/javascript';
-hljs.registerLanguage('javascript', javascript);
+### Self Hosting
+
+The [download page][5] can quickly generate a custom bundle including only the languages you need.
+
+Alternatively, you can build a browser package package from [source](#source):
```
Comment (allejo) [inline README.md]:
```suggestion
Alternatively, you can build a browser package from [source](#source):
```

File: README.md [hunk 3]
```diff
@@ -153,64 +180,95 @@ both AMD and CommonJS, so if you wish you can use RequireJS or
...
-import hljs from 'highlight.js/lib/core';
-import 'highlight.js/styles/github.css';
+**Note:** Building from source should always result in the smallest size builds.  Thw website download page is optimized for speed, not size.
+
+
+#### Prebuilt CDN assets
+
+You can also download and self-host the same assets we serve up via our own CDNs.  We publish those builds to the [cdn-release](https://github.com/highlightjs/cdn-release) GitHub repository.  You can easily pull individual files off the CDN endpoints with `curl`, etc; if say you only needed `highlight.min.js` and a single CSS file.
+
+There is also an npm package [@highlightjs/cdn-assets](https://www.npmjs.com/package/@highlightjs/cdn-assets) if pulling the assets in via `npm` would be easier for your build process.
```
Comment (allejo) [inline README.md]:
```suggestion
There is also an npm package [@highlightjs/cdn-assets](https://www.npmjs.com/package/@highlightjs/cdn-assets) if pulling the assets in via `npm` or `yarn` would be easier for your build process.
```

File: README.md [hunk 4]
```diff
@@ -153,64 +180,95 @@ both AMD and CommonJS, so if you wish you can use RequireJS or
...
+
+
+### NPM / Node.js server module
+
+Highlight.js can also be used on the server. The package with all supported languages can be installed from NPM:
```
Comment (allejo) [inline README.md]:
```suggestion
Highlight.js can also be used on the server. The package with all supported languages can be installed from NPM or Yarn:
```

File: README.md [hunk 5]
```diff
@@ -153,64 +180,95 @@ both AMD and CommonJS, so if you wish you can use RequireJS or
...
+
+```
+npm install highlight.js
+```
```
Comment (allejo) [inline README.md]:
~~~suggestion
```bash
npm install highlight.js

# or

yarn add highlight.js
```
~~~

File: README.md [hunk 6]
```diff
@@ -153,64 +180,95 @@ both AMD and CommonJS, so if you wish you can use RequireJS or
...
+
+Alternatively, you can build it from [source](#source):
+
+```
```
Comment (allejo) [inline README.md]:
~~~suggestion
```bash
~~~

**Comment by yyyc514** [comment]
Thanks, accepted your changes.

**Pull request merged**